Makefile

```diff
@@ -0,0 +1,53 @@
+# **************************************************************************** #
+#                                                                              #
+#                                                         :::      ::::::::    #
+#    Makefile                                           :+:      :+:    :+:    #
+#                                                     +:+ +:+         +:+      #
+#    By: jteissie <jteissie@student.42.fr>          +#+  +:+       +#+         #
+#                                                 +#+#+#+#+#+   +#+            #
+#    Created: 2024/08/14 18:14:24 by jteissie          #+#    #+#              #
+#    Updated: 2024/08/14 18:16:09 by jteissie         ###   ########.fr        #
+#                                                                              #
+# **************************************************************************** #
+
+NAME	= cub3d
+CC		= cc
+CFLAGS	= -Wall -Werror -Wextra -g3
+
+SRCDIR	= srcs
+OBJDIR	= obj
+CFILES	= main.c
+
+INCS	=	-I ./include \
+			-I ./libft
+
+vpath %.c ./ srcs/
```

Comment:
Using vpath could have potential risks if more than one source file have the same name in different repo.